AMW_angular/io/src/app/servers/servers-filter/servers-filter.component.html

```diff
@@ -0,0 +1,69 @@
+<div class="container">
+    <div class="card">
+        <div class="card-header">
+            Filter
+        </div>
+        <div class="card-body">
+            <div class="row align-items-center mb-3">
+                <div class="col-sm-1">
+                    <label for="appServer" class="form-label">AppServer</label>
+                </div>
+                <div class="col-sm-3">
+                    <input type="text" class="form-control" id="appServer"
+                           [ngbTypeahead]="search"
+                           [(ngModel)]="appServer">
+                </div>
+            </div>
+            <div class="row align-items-center mb-3">
+                <div class="col-sm-1">
+                    <label class="form-label">Runtime</label>
+                </div>
+                <div class="col-sm-3">
+                    <select class="form-select" [(ngModel)]="selectedRuntimeName">
+                      <option [value]="'All'">All</option>
+                      @for(runtime of runtimes(); track runtime.id){
+                        <option [value]="runtime.name">{{ runtime.name }}</option>
+                      }
+                    </select>
+                </div>
+            </div>
+            <div class="row align-items-center mb-3">
+                <div class="col-sm-1">
+                    <label class="form-label">Env</label>
+                </div>
+                <div class="col-sm-3">
+                    <select class="form-select" [(ngModel)]="selectedEnvironmentName">
+                        <option [value]="'All'">All</option>
+                        @for(env of environments(); track env.id){
+                            <option [value]="env.name">{{ env.name }}</option>
+                        }
+                    </select>
+                </div>
+            </div>
+            <div class="row align-items-center mb-3">
+                <div class="col-sm-1">
+                    <label for="host" class="form-label">Host</label>
+                </div>
+                <div class="col-sm-3">
+                    <input type="text" class="form-control" id="host" [(ngModel)]="host">
+                </div>
+            </div>
+            <div class="row align-items-center mb-3">
+                <div class="col-sm-1">
+                    <label for="node" class="form-label">Node</label>
+                </div>
+                <div class="col-sm-3">
+                    <input type="text" class="form-control" id="node" [(ngModel)]="node">
+                </div>
+            </div>
+        </div>
+        <div class="card-footer">
+            <div class="row justify-content-between">
```

Comment:
Buttons could be align next to each other on the right? Would be similar to apps-page and modals